docs/user/adding-glean-to-your-project.md

```diff
@@ -111,18 +111,73 @@ github "mozilla/glean" "master"
 
 </div>
 
+<div data-lang="Python" class="tab">
+
+It is recommended that you use a virtual environment for your work to isolate the dependencies for your project. There are many popular abstractions on top of virtual environments in the Python ecosystem which should help manage your project dependencies.
```

Comment:
@hackebrot : Curious what you think of this.  I don't want to write a tutorial on virtual environments (and all the related tools) here -- but should we recommend anything in particular?